types/stylelint/index.d.ts

```diff
@@ -160,6 +160,7 @@ declare module 'stylelint' {
 		export type Rule<P = any, S = any> = RuleBase<P, S> & {
 			ruleName: string;
 			messages: RuleMessages;
+			meta: { url: string; deprecated?: boolean };
```

Comment:
[ask] Since the next version, `rule.meta` would be required. Is this too strict for plugin authors?

Reply:
I think it should be opt-in, but recommended in the docs.

Reply:
@jeddy3 Thanks for the quick feedback. Marked as optional. (6afacb30)